app/views/classified_listings/index.html.erb

```diff
@@ -32,15 +32,15 @@
     data-allcategories="<%= categories_for_display.to_json %>"
     <% if @displayed_classified_listing %>
       data-displayedlisting="<%= @displayed_listing_json %> "
-    <% end %>
-  >
+    <% end %>>
+
     <div class="classified-filters">
     <div class="classified-filters-categories">
-      <a href="/listings" class="<%= "selected" if params[:category].blank? %> data-no-instant="true">all</a>
+      <a href="<%= classified_listings_path %>" class="<%= "selected" if params[:category].blank? %> data-no-instant=" true ">all</a>
```

Comment:
@rhymes Good idea using the path helpers, changed now.

Reply:
why the spaces around `true` ?

Reply:
I don't now. Didn't even notice them, don't know where they came from. I'll remvoe them.